### DISCUSSION TITLE
i just want to use it in windows,but install some not work,please help me

### DISCUSSION THREAD

**Discussion by 1096015776** [mason-org/mason.nvim#689] [Q&A]
when i use mason install some npm lsp, but i got some error, i checkhealth i found npm not available,but i sure i have it,and i can run 'npm --version' in anywhere, but in powershell i got '/c is a directory' maybe i should give up in windows,but i really want to get it

**Comment by ca-mantis-shrimp** [comment]
Same issue on windows. LazyVim seems to work fine for jsonls, but on windows machines I'm unable to download it because it's running into an issue with NPM.

What's strange is while checkhealth says it simply can't find npm, when I look at the lsp log it seems to show that the npm command ran but just outputs the help message indicating an error, I also sometimes get an "error 1, signal 0" when trying to install jsonls
